src/azure-cli/azure/cli/command_modules/vm/manual/_params.py

```diff
@@ -19,3 +22,37 @@ def load_arguments(self, _):
         c.argument('resource_group_name', resource_group_name_type)
         c.argument('ssh_public_key_name', options_list=['--name', '-n', '--ssh-public-key-name'], type=str, help='The '
                    'name of the SSH public key.', id_part='name')
+
+    with self.argument_context('sig group-list') as c:
+        c.argument('shared_to', options_list=['--scope'], arg_type=get_enum_type(['tenant', 'subscription']),
+                   help='The query parameter to decide what shared galleries to fetch when doing listing operations.',
+                   default='subscription')
+
+    for scope in ['sig share add', 'sig share remove']:
+        with self.argument_context(scope) as c:
+            c.argument('resource_group_name', resource_group_name_type)
+            c.argument('gallery_name', type=str, help='The name of the Shared Image Gallery.', id_part='name')
+            c.argument('subscription_ids', nargs='+', help='A list of subscription ids to share the gallery.')
+            c.argument('tenant_ids', nargs='+', help='A list of tenant ids to share the gallery.')
+
+    with self.argument_context('sig share add') as c:
+        c.argument('op_type', default='Add', deprecate_info=c.deprecate(hide=True),
+                   help='distinguish add operation and remove operation')
+
+    with self.argument_context('sig share remove') as c:
+        c.argument('op_type', default='Remove', deprecate_info=c.deprecate(hide=True),
+                   help='distinguish add operation and remove operation')
```

Comment:
May I ask why we need to hide these two new parameters?

Reply:
With command `sig share add`, op_type is `Add`.
With command `sig share remove`, op_type is `Remove`.
By setting the argument `op_type` a default value and hidden from user, 2 commands can share the same custom function because the 2 operation has the same API and parameters.

Reply:
@xfz11 for more information

Reply:
Got it, thanks~